browser/resources/settings/brave_overrides/autofill_page.ts

```diff
@@ -6,22 +6,40 @@
 import {
   html,
   RegisterPolymerTemplateModifications,
-  RegisterPolymerComponentReplacement
+  RegisterPolymerComponentReplacement,
+  RegisterPolymerPrototypeModification
 } from 'chrome://resources/brave/polymer_overriding.js'
 import {
   BraveSettingsAutofillPageElement
 } from '../brave_autofill_page/brave_autofill_page.js'
-import {loadTimeData} from '../i18n_setup.js'
-import '../email_aliases_page/email_aliases_page.js'
+import { loadTimeData } from '../i18n_setup.js'
+import '../email_aliases_page/email_aliases_section.js'
+import { routes } from '../route.js'
+import type { Route } from '../router.js'
 
 RegisterPolymerComponentReplacement(
   'settings-autofill-page', BraveSettingsAutofillPageElement
 )
 
+RegisterPolymerPrototypeModification({
+  'settings-autofill-page-index': (prototype) => {
+    const original = prototype.currentRouteChanged
+    prototype.currentRouteChanged =
+      function (newRoute: Route, oldRoute?: Route) {
+      if (typeof original === 'function') {
+        original.call(this, newRoute, oldRoute)
+      }
+
+      if (newRoute === routes.EMAIL_ALIASES) {
+        this.$.viewManager.switchView('email-aliases', 'no-animation',
+                                      'no-animation')
+      }
```

Comment:
Hey, if we match this route lets return and not call the super function. That way if upstream adds a default case this won't break.
```suggestion
      if (newRoute === routes.EMAIL_ALIASES) {
        this.$.viewManager.switchView('email-aliases', 'no-animation',
                                      'no-animation')
        return
      }
      original.call(this, newRoute, oldRoute)
```